### DISCUSSION TITLE
Wireguard and OpenVPN on the same subnet

### DISCUSSION THREAD

**Discussion by learningubuntu** [pivpn/pivpn#1778] [Support]
I will try and explain my dilema.

I have wireguard installed and set up (with circa 10 devices) and its working perfectly, however, i would like to add a few devices that only support OpenVPN. 

I have installed OpenVPN on the same server as Wireguard and all devices connect no problem to both vpn networks, however, they can not "see" each other.

Is there any way I can get both VPN servers to opperate on the same subnet? Alternatively, can anyone give some pointers on how to route traffic between the two networks. Apologies if this is a easy fix, i have looked but just cant find anything and cant figure out how to do it.

Thank you in advance

**Comment by orazioedoardo** [comment]
You could add routes to each other network using the Raspberry Pi as a gateway. You can use the OS features to do that. OpenVPN also has an option to be added in the client to do this: https://man.archlinux.org/man/openvpn.8#route
